.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "arrowParens": "avoid",
+  "singleQuote": true,
+  "tabWidth": 2,
+  "semi": true,
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false
```

Comment:
add that property also where the brackets are not removed from arguements of arrow function when there is just one sinlgle aruements.
Correct: const fn = (a) => { return b;}

Wrong: const fn = a => {return b;}

Reply:
I would suggest just glancing over the first line of the [description of eslint-config-prettier](https://github.com/prettier/eslint-config-prettier). Your concern is taken care of :slightly_smiling_face: . 
Also,
according to our .prettierrc:
```js
wrong: const fn = (a) => { return b;}
correct: const fn = a => {return b;}
```